### ISSUE TITLE
not convergence

### ISSUE THREAD

**Issue by gmt710** [BIGBALLON/cifar-10-cnn#12]
Network_in_Network_keras.py
Trainning accuracy is always around 1.0%.

**Comment by BIGBALLON** [comment]
@gmt710 Can you show your code for me? Which can help me to find your problems.
BTW: the worst accuracy is around 10%. so why you get 1.0%, that is impossible.
